simple_history/tests/tests/test_admin.py

```diff
@@ -467,7 +468,9 @@ def test_history_form_view_without_getting_history(self):
             'save_on_top': admin.save_on_top,
             'root_path': getattr(admin_site, 'root_path', None),
         }
+        extra_context = {}
```

Comment:
Why do you need to set `extra_context = {}` here?

Reply:
in the test it is not necessary.

Reply:
Shouldn't the test pass with `extra_context=None` as the default? Or do you want to set the default to `extra_context={}`?

Reply:
In the test, I want to set the default extra_context={}
Your option will work too. extra_context in the test does not affect the functionality

Reply:
Won't the test still pass if you don't have this line? Essentially this makes line 473 irrelevant.

Reply:
I add extra_context in context. In this case, I do not pass the custom parameters and extra_context = {} can be omitted. Should I delete this line in tests?

Reply:
Yes delete this line and then I'll approve and merge

Reply:
Done :)